javascript-local-storage/data.js

```diff
@@ -0,0 +1,12 @@
+/* exported todos */
+window.addEventListener('beforeunload', unloadHandler);
+function unloadHandler(event) {
+  event.preventDefault();
```

Comment:
You don't want to do this, I think it will try to prevent the user from closing the page